src/modules/launcher/Plugins/Microsoft.Plugin.Program/Logger/ProgramLogger.cs

```diff
@@ -17,6 +17,31 @@ namespace Microsoft.Plugin.Program.Logger
     /// </summary>
     internal static class ProgramLogger
     {
+        /// <summary>
```

Comment:
Added Warn Method which could be used for non-fatal index related issues (e.g. Not found / not accessible / not parse able)